plugin/trino-hive/src/test/java/io/trino/plugin/hive/AbstractTestHive.java

```diff
@@ -652,7 +652,7 @@ public void setupClass()
         executor = newCachedThreadPool(daemonThreadsNamed("hive-%s"));
         heartbeatService = newScheduledThreadPool(1);
         // Use separate staging directory for each test class to prevent intermittent failures coming from test parallelism
-        temporaryStagingDirectory = createTempDirectory("trino-staging-");
+        temporaryStagingDirectory = Paths.get("/tmp/" + UUID.randomUUID());
```

Comment:
Looks fine - unless this sometimes maps to local file system. Then it is suboptimal. Maybe the path should be provided from the `run_hive_*_test.sh` script?

Reply:
add a commentm why, so it;s not tempting to "fix" this code in the future